.github/workflows/build.yml

```diff
@@ -23,6 +23,7 @@ jobs:
     - name: Build Only
       uses: docker/bake-action@v6
```

Comment:
As long as we are modifying this, I think the new best practice is to use the sha instead of a tag
`uses: docker/bake-action@5ca506d06f70338a4968df87fd8bfee5cbfb84c7 # v6`
I believe dependabot will then use the sha going forward. Of course this does not mean dependabot will read the release warnings!
What do you think?